### PR TITLE
Spelling rework

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -56,70 +56,70 @@
...
     <string name="text_storagePath">Storage path</string>
     <string name="text_storagePathSummary">New files will be saved to this folder</string>
     <string name="mesg_pathUnavailable">%s currently unavailable</string>
-    <string name="butn_showOnFileManager">Show on file manager</string>
+    <string name="butn_showOnFileManager">Show in file manager</string>
     <string name="text_chooseFileManager">Choose file manager</string>
-    <string name="text_ipAddress">IP Address</string>
-    <string name="text_allowToAccess">Allow to access</string>
+    <string name="text_ipAddress">IP address</string>
+    <string name="text_allowToAccess">Allow access to</string>
```
Comment (velitasali) [inline app/src/main/res/values/strings.xml]: I think in this context, previous one was better.
screenshot_20180114-180234

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -56,70 +56,70 @@
...
     <string name="text_empty">(empty)</string>
     <string name="text_textStream">Text stream</string>
     <string name="butn_wifiSettings">Wi-Fi</string>
     <string name="text_pendingTransfers">Pending transfers</string>
     <string name="text_stopping">Stopping!</string>
-    <string name="text_cancellingTransfer">The job is being cancelled</string>
+    <string name="text_cancellingTransfer">Canceling job…</string>
     <string name="butn_killNow">Kill now</string>
-    <string name="mesg_groupOngoingNotice">This file will be received after %s</string>
+    <string name="mesg_groupOngoingNotice">File complete in %s</string>
```
Comment (velitasali) [inline app/src/main/res/values/strings.xml]: %s is converted to the name of the file which is being received. So I think we should leave as it is.

**Comment by comradekingu** [comment]
@velitasali It is possible to misinterpret the placeholder as a timestamp, so i rewrote it to avoid errors in translation.
I don't understand what "Allow to access" means or what it does.

**Comment by velitasali** [comment]
@comradekingu It is "Allow it to access this device" in full sentence. I think the grammer is okay but I don't know about the meaning. We can change it to "allow to connect" or something like that. Thank you for your work by the way 👍

**Comment by comradekingu** [comment]
@velitasali "It" being TrebleShot?

**Comment by velitasali** [comment]
@comradekingu the other device

**Pull request merged**